File: backend/src/main/java/com/ladysparks/ttaenggrang/domain/stock/dto/StudentStockDTO.java
package com.ladysparks.ttaenggrang.domain.stock.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class StudentStockDTO {

    private Long stockId;               // 주식 ID
    private String stockName;           // 주식명
    private int quantity;               // 보유 주식 수
    private int purchasePrice;          // 최초 구매 가격
    private LocalDateTime purchaseDate; // 최초 구매 날짜
    private int currentPrice;           // 현재 주가

}

File: frontend/Ttaenggrang/app/src/main/java/com/ladysparks/ttaenggrang/data/model/dto/StudentStockDto.kt
package com.ladysparks.ttaenggrang.data.model.dto

data class StudentStockDto(
    val stockId: Int,          // 보유 주식 ID
    val stockName: String,     // 주식 이름
    val ownedQty: Int,         // 보유 수량
    val purchasePrice: Int,    // 매입 가격 (매입 당시
    val purchaseDate: String,  // 매입 날짜 (ISO 8601 형식: yyyy-MM-dd'T'HH:mm:ss)
    val currentPrice: Int      // 현재 가격
)
